.changes/8e87c6dd-5138-4b79-a0c2-255ecf31898b.json

```diff
@@ -0,0 +1,6 @@
+{
+    "id": "8e87c6dd-5138-4b79-a0c2-255ecf31898b",
+    "type": "misc",
+    "description": "Upgrade to latest versions of OkHttp, Okio, Kotlin",
```

Comment:
**Question**: Is there value in creating a breaking change discussion for this?

Reply:
I don't really think so. The OkHttp breaking changes are broad and we can't easily write a migration guide for each possible case.

Reply:
I think it's worth mentioning it and maybe not getting into too much detail about it. Like Matas said the changes are broad. This will break customers not using the latest ok http version?